app/org/sagebionetworks/bridge/models/studies/StudyParticipant.java

```diff
@@ -45,7 +45,9 @@ public Boolean getNotifyByEmail() {
         return (emptyString == null) ? null : Boolean.valueOf(emptyString);
     }
     public void setNotifyByEmail(Boolean notifyByEmail) {
-        put(UserProfile.NOTIFY_BY_EMAIL_FIELD, notifyByEmail.toString());
+        if (notifyByEmail != null) {
+            put(UserProfile.NOTIFY_BY_EMAIL_FIELD, notifyByEmail.toString());    
+        }
```

Comment:
Yes this could be called with null (in an unusual user state, probably only an artifact of testing and not something that happens in production), and we wouldn't consider null the equivalent of being false, so we'd just leave it out of the map entirely.